### PR TITLE
Editorial: register application/manifest+json with IANA

### LINKED ISSUE #970: Register media type with IANA
The `application/manifest+json` media type should be registered with IANA "[o]ne to two months prior to Candidate Recommendation" (https://www.w3.org/2020/01/registering-mediatypes).

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #970 

This change (choose at least one, delete ones that don't apply):

* Makes editorial changes (changes informative sections, or changes normative sections without changing behavior)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/pull/985.html" title="Last updated on Jun 28, 2021, 10:00 AM UTC (28db2a2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/985/f9da05d...28db2a2.html" title="Last updated on Jun 28, 2021, 10:00 AM UTC (28db2a2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2525,175 +2527,19 @@ <h2>
         to mimic the user interface of another application.
       </p>
     </section>
-    <section>
+    <section class="appendix">
       <h2>
-        IANA considerations
+        The `application/manifest+json` MIME type
```
Comment (christianliebel) [inline index.html:2532]:
```suggestion
        The `application/manifest+json` media type
```

File: index.html [hunk 2]
```diff
@@ -2525,175 +2527,19 @@ <h2>
...
       </h2>
       <p>
-        The following registrations are for community review and will be
-        submitted to the <a href="https://www.ietf.org/iesg/">IESG</a> for
-        review, approval, and registration with <a href=
-        "https://www.iana.org/">IANA</a>.
+        The <code><dfn data-local-lt=
+        "application manifest MIME type">application/manifest+json</dfn></code>,
```
Comment (christianliebel) [inline index.html:2536]:
```suggestion
        "application manifest media type">application/manifest+json</dfn></code>,
```

**Review by christianliebel — CHANGES_REQUESTED** [review]
Very cool! Thanks a lot.

Just a nit: The new name for "MIME type" seems to be "media type," at least the [IANA website](https://www.iana.org/assignments/media-types/media-types.xhtml) states that.

**Comment by marcoscaceres** [comment]
Good point.. fixing!

**Review by christianliebel — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Had to add the registration back in. Apparently, we also need to have it in the spec, although redundant.

**Pull request merged**